docs/release-process/instructions.md

```diff
@@ -0,0 +1,92 @@
+# How to use the release tooling
+
+## Creating the tracking issue
+
+First, create an issue on the microsoft/go repository for release progress updates. When you pass the issue number to the release automation pipeline, that makes the pipeline post a comment on that issue to notify you when a step completes or fails.
+
+If dealing with multiple releases at the same time (for example, 1.17 and 1.18 security patches released on the same day), create one issue per version number. This makes it easier to keep track of each release's progress as an outside observer.
+
+> The "release-start" pipeline is planned to create these issues automatically for common servicing events. (See [README.md](README.md).)
+
+## Upstream release
+
+After an upstream release, or in anticipation of an upstream release, go to the [`microsoft-go-infra-release-build`](https://dev.azure.com/dnceng/internal/_build/results?buildId=1780270&view=results) pipeline. Press "Run new" and fill in the blanks in the popup.
+
+For example, if upstream is releasing 1.42.3, the parameters might look like this:
+
+> ![](images/run-upstream-release.png)
```

Comment:
These don't show up on the github preview, is that just an artifact of the rendering?

Reply:
Hmm, looks like they don't show up in the "rich diff" mode, but they do show up at "view file". Strange, don't think I've noticed that before. Seems like GitHub could have at least shown some indication that there's an image if there's some good reason they can't simply show it inline. 😕